handlers/meshery_application_handler.go

```diff
@@ -109,18 +109,6 @@ func (h *Handler) handleApplicationPOST(
 
 	// If Content is not empty then assume it's a local upload
 	if parsedBody.ApplicationData != nil {
-		applicationName, err := models.GetApplicationName(parsedBody.ApplicationData.ApplicationFile)
-		if err != nil {
-			obj := "save"
-			h.log.Error(ErrApplicationFailure(err, obj))
-			http.Error(rw, ErrApplicationFailure(err, obj).Error(), http.StatusBadRequest)
-			return
-		}
-
-		// Assign a name if no name is provided
-		if parsedBody.ApplicationData.Name == "" {
-			parsedBody.ApplicationData.Name = applicationName
-		}
```

Comment:
We have to make sure as the client that we are passing name field. @piyushsingariya 
like: 
```js
{
	"application_data": {
		"name": "Any name",
		"application_file": ".....",
		"format": "pattern" // default is pattern
	}
}
```

Reply:
Yes that is right as we can't extract name from K8s manifest